tests/jenkins/jobs/uploadMinSnapshotsToS3_Jenkinsfile.txt

```diff
@@ -32,6 +32,7 @@ ccc
                   uploadMinSnapshotsToS3.sh(
         cp -v tests/data/tar/builds/opensearch/dist/opensearch-min-1.2.2-SNAPSHOT-linux-x64.tar.gz tests/data/tar/builds/opensearch/dist/opensearch-min-1.2.2-SNAPSHOT-linux-x64-latest.tar.gz
         cp -v tests/data/tar/builds/opensearch/dist/opensearch-min-1.2.2-SNAPSHOT-linux-x64.tar.gz.sha512 tests/data/tar/builds/opensearch/dist/opensearch-min-1.2.2-SNAPSHOT-linux-x64-latest.tar.gz.sha512
+        cp -v tests/data/tar/builds/opensearch/dist/../manifest.yml tests/data/tar/builds/opensearch/dist/opensearch-min-1.2.2-SNAPSHOT-linux-x64-latest.tar.gz.build-manifest.yml
```

Comment:
This is a dist manifest from source copying Can you keep is just manifest instead of build-manifest. 
Sorry about that just realized

Reply:
It is build manifest not dist manifest.